### PR TITLE
Support arm machines

### PR DESCRIPTION
**Pull request by moadqassem**
**What this PR does / why we need it**:
Adding ARM support for worker nodes created by the machine controller.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged)*:
Fixes kubermatic/kubermatic#6252

**Special notes for your reviewer**:

**Optional Release Note**:
```release-note
ARM worker nodes support 
```

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -217,6 +217,28 @@ presubmits:
               memory: 1Gi
               cpu: 500m
 
+  - name: pull-machine-controller-e2e-aws
```
Comment (xmudrii) [inline .prow.yaml:220]: There is a job with the same name defined above, that's why the presubmit fails.

File: .prow.yaml [hunk 2]
```diff
@@ -217,6 +217,28 @@ presubmits:
...
+    error_on_eviction: true
+    clone_uri: "ssh://git@github.com/kubermatic/machine-controller.git"
+    labels:
+      preset-aws: "true"
+      preset-hetzner: "true"
+      preset-e2e-ssh: "true"
+      preset-rhel: "true"
+    spec:
+      containers:
+        - image: golang:1.13.8
```
Comment (xmudrii) [inline .prow.yaml:232]: We have bumped Go to 1.15.3 in #853. Make sure to rebase the PR and update the image for this job.

**Review by xmudrii — COMMENTED** [review]
Left two nits, but other than that looks good to me.

**Comment by moadqassem** [comment]
@xmudrii PTAL

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**